azure-spring-boot-cloudfoundry/pom.xml

```diff
@@ -0,0 +1,226 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.microsoft.azure</groupId>
+        <artifactId>azure-spring-boot-parent</artifactId>
+        <version>2.1.8-SNAPSHOT</version>
+        <relativePath>../azure-spring-boot-parent/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>azure-spring-boot-cloudfoundry</artifactId>
+    <packaging>jar</packaging>
+
+    <dependencies>
```

Comment:
Let's review the dependencies. I think not of all them are needed in this module

Reply:
I've remove them and i left the required

Reply:
I still see many dependencies may not needed here. I think `org.springframework.boot:spring-boot-autoconfigure` should be enough with the test library. Let's take a look into it again please.